stamp_generation/stamp_templates/stamp_base_template.jj2

```diff
@@ -51,7 +51,7 @@
                 {% if contents[page.media_index].content_type.name == 'EMBEDDED_PINTEREST_PIN'-%}
                     {%- include "embedded_pinterest_pin_template.jj2" -%}
                 {%- endif -%}
-                {% if not page.is_embedded_content %}
+                {% if not (page.is_embedded_content or contents[page.media_index].content_type.name == 'QUOTE') %}
```

Comment:
Its better to use functions. We should minimize logic in jinja files.

Reply:
Makes sense! Thanks for suggestion. Will be updated in next iterations. Merging this PR.